File: primer-curso/programacion/unidad-02-Java&Python/python/PrintArrayDinamico.py
numeros = [6,3,2,7,1,9]

print(f"Array de numeros: {numeros}")


for i in range(0,len(numeros)):
	num = ""
	for j in range(0,numeros[i]):
		num+= str(numeros[i])
	print(f"{num}")

File: primer-curso/programacion/unidad-02-Java&Python/java/PrintArrayDinamico.java
public class PrintArrayDinamico{
	public static void main(String[] args){

		String texto = "";

		int[] numeros = {2,4,6,1,7};

		for(int i = 0; i < numeros.length; i++){
			for(int j = 0; j < numeros[i]; j++){
				System.out.print(numeros[i] + " ");
			}
			System.out.println();
		}
	}
}
